src/core/Akka.Cluster.Tests/ClusterConfigSpec.cs

```diff
@@ -23,40 +25,49 @@ public class ClusterConfigSpec : AkkaSpec
-            Assert.Equal(TimeSpan.FromSeconds(1), settings.HeartbeatInterval);
-            Assert.Equal(5, settings.MonitoredByNrOfMembers);
-            Assert.Equal(TimeSpan.FromSeconds(5), settings.HeartbeatExpectedResponseAfter);
-            Assert.Equal(TimeSpan.FromSeconds(1), settings.LeaderActionsInterval);
-            Assert.Equal(TimeSpan.FromSeconds(1), settings.UnreachableNodesReaperInterval);
-            Assert.Null(settings.PublishStatsInterval);
-            Assert.Null(settings.AutoDownUnreachableAfter);
-            Assert.Equal(1, settings.MinNrOfMembers);
-            //TODO:
-            //Assert.AreEqual(ImmutableDictionary.Create<string, int>(), settings.);
-            Assert.Equal(ImmutableHashSet.Create<string>(), settings.Roles);
-            Assert.Equal("akka.cluster.default-cluster-dispatcher", settings.UseDispatcher);
-            Assert.Equal(.8, settings.GossipDifferentViewProbability);
-            Assert.Equal(400, settings.ReduceGossipDifferentViewProbability);
-            Assert.Equal(TimeSpan.FromMilliseconds(33), settings.SchedulerTickDuration);
-            Assert.Equal(512, settings.SchedulerTicksPerWheel);
-            Assert.True(settings.MetricsEnabled);
-            //TODO:
-            //Assert.AreEqual(typeof(SigarMetricsCollector).FullName, settings.MetricsCollectorClass);
-            Assert.Equal(TimeSpan.FromSeconds(3), settings.MetricsGossipInterval);
-            Assert.Equal(TimeSpan.FromSeconds(12), settings.MetricsMovingAverageHalfLife);
-            Assert.Equal(false, settings.VerboseHeartbeatLogging);
+            settings.LogInfo.Should().BeTrue();
+
+            settings.SeedNodes.Should().BeEquivalentTo(ImmutableList.Create<Address>());
+            settings.SeedNodeTimeout.Should().Be(5.Seconds());
+            settings.RetryUnsuccessfulJoinAfter.Should().Be(10.Seconds());
+            settings.PeriodicTasksInitialDelay.Should().Be(1.Seconds());
+            settings.GossipInterval.Should().Be(1.Seconds());
+            settings.GossipTimeToLive.Should().Be(2.Seconds());
+            settings.HeartbeatInterval.Should().Be(1.Seconds());
+            settings.MonitoredByNrOfMembers.Should().Be(5);
+            settings.HeartbeatExpectedResponseAfter.Should().Be(1.Seconds());
+            settings.LeaderActionsInterval.Should().Be(1.Seconds());
+            settings.UnreachableNodesReaperInterval.Should().Be(1.Seconds());
+            settings.PublishStatsInterval.Should().NotHaveValue();
+            settings.AutoDownUnreachableAfter.Should().NotHaveValue();
+            // settings.DownRemovalMargin.Should().Be(TimeSpan.Zero);
+            settings.MinNrOfMembers.Should().Be(1);
+            settings.MinNrOfMembersOfRole.Should().Equal(ImmutableDictionary<string, int>.Empty);
+            settings.Roles.Should().BeEquivalentTo(ImmutableHashSet<string>.Empty);
+            // settings.UseDispatcher.Should().Be(Dispatchers.DefaultDispatcherId);
```

Comment:
Why we have a different dispatcher for Cluster, while JVM versions uses the same default dispatcher everywhere?